install/installer/traf_sqgen

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+# @@@ START COPYRIGHT @@@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# @@@ END COPYRIGHT @@@
+
+export TRAF_CONFIG=/etc/trafodion/trafodion_config
+
+source $TRAF_CONFIG
+
+STARTING_NODE=$(hostname)
+
+#==========================================
+echo "***INFO: starting sqgen" | tee -a $INSTALL_LOG
+cd $SQ_ROOT/sql/scripts
+$SQ_ROOT/sql/scripts/sqgen
+SQGEN_RC=$?
+if [ "$SQGEN_RC" != "0" ]; then
+    echo "***ERROR: sqgen failed with RC=$SQGEN_RC. Check install log file for details." | tee -a $INSTALL_LOG
+    exit -1;
+fi
+#==========================================
+
+if [ "$node_count" -ne "1" ]; then
```

Comment:
Should this now be all_node_count instead of node_count?

 It would have been better if node_count was assigned all_node_count  and MY_NODES was assigned ALL_NODES. That way changes could be kept to  minimal